app/controllers/teammates_controller.rb

```diff
@@ -0,0 +1,58 @@
+class TeammatesController < ApplicationController
+  before_action :require_invitation
+  before_action :require_pending
+  rescue_from ActiveRecord::RecordNotFound, :with => :incorrect_token
+
+  def accept
+    if !current_user
```

Comment:
I'm having issues making that change and I saw this comment after a major rebase into master. So for now I'm leaving this.